### ISSUE TITLE
Typo in docs/core/howto/logger.rst

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7944]
|[<img alt="hynek's avatar" src="https://avatars.githubusercontent.com/u/41240?s=50" width="50" height="50">](https://github.com/hynek)| @hynek reported|
|-|-|
|Trac ID|trac#7944|
|Type|defect|
|Created|2015-06-18 14:01:32Z|
|Branch|https://github.com/twisted/twisted/tree/doc-howto-logger-typo-7944|

Line 119: “This can be used **my** observers to obtain a traceback”

Attachments:

* [logger-typo.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/a12/a128426846ae352e8649434efb8dc32db141da8d/da541cbdafcfe3c86bb305251e5951f03030f51e.patch) (2691 bytes) - added by thijs on 2015-11-18 16:50:18Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__7944 7944
type__defect defect
reporter__hynek hynek
priority__normal normal
milestone__ 
branch__branches_doc_howto_logger_typo_7944 branches/doc-howto-logger-typo-7944
branch_author__adiroiban adiroiban
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1434636092090653 1434636092090653
changetime__1447928657700382 1447928657700382
version__None None
owner__thijs thijs
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_1"></a>|@thijstriemstra commented|
|-|-|

Attached patch fixes the typo (and some others).

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_2"></a>|@adiroiban commented|
|-|-|

(In [46260]) Branching to doc-howto-logger-typo-7944.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban set owner to @thijstriemstra|
|-|-|

Many thanks for your contribution!
Changes looks good.

I have build the docs on my system and all is ok.

Will sent to test just to have a run before merge and if all is ok (as it should be) I will merge.

Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_4"></a>|@adiroiban set status to `closed`|
|-|-|

(In [46285]) Merge doc-howto-logger-typo-7944: Fix logger doc typos.

Author: thijs
Reviewer: adiroiban
Fixes: [#7944](https://github.com/twisted/twisted/issues/7944)
